arithmetization/src/test/java/net/consensys/linea/zktracer/module/mxp/MxpTest.java

```diff
@@ -41,9 +41,12 @@
 import org.hyperledger.besu.datatypes.Wei;
 import org.hyperledger.besu.ethereum.core.Transaction;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
 
 // https://github.com/Consensys/linea-besu-plugin/issues/197
 
+@Execution(ExecutionMode.SAME_THREAD)
 public class MxpTest {
```

Comment:
For some reason one of the MxpTests fails if run concurrently.

Reply:
could be because of https://github.com/Consensys/linea-tracer/pull/1120, fixed now with latest arith-dev